<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Photo_Editor</class>
 <widget class="QMainWindow" name="Photo_Editor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Инструменты</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_brush_2"/>
    <addaction name="grain"/>
    <addaction name="pip_2"/>
    <addaction name="separator"/>
    <addaction name="action_line_2"/>
    <addaction name="circle"/>
    <addaction name="action_square"/>
    <addaction name="separator"/>
    <addaction name="canv_col"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Цвет</string>
    </property>
    <addaction name="color"/>
    <addaction name="separator"/>
    <addaction name="red"/>
    <addaction name="orange"/>
    <addaction name="yellow"/>
    <addaction name="green"/>
    <addaction name="light_blue"/>
    <addaction name="blue"/>
    <addaction name="violet"/>
    <addaction name="separator"/>
    <addaction name="black"/>
    <addaction name="white"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Толщина</string>
    </property>
    <addaction name="small"/>
    <addaction name="med"/>
    <addaction name="big"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="op"/>
    <addaction name="sav"/>
    <addaction name="saas"/>
   </widget>
   <addaction name="menu_4"/>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <action name="action_brush">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Кисть</string>
   </property>
  </action>
  <action name="action_line">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Линия</string>
   </property>
  </action>
  <action name="action_3">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Окружность</string>
   </property>
  </action>
  <action name="action_brush_2">
   <property name="icon">
    <iconset>
     <normaloff>icons/инструменты/кисть.png</normaloff>icons/инструменты/кисть.png</iconset>
   </property>
   <property name="text">
    <string>Кисть</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>Размер</string>
   </property>
  </action>
  <action name="action_line_2">
   <property name="icon">
    <iconset>
     <normaloff>icons/инструменты/линия.png</normaloff>icons/инструменты/линия.png</iconset>
   </property>
   <property name="text">
    <string>Линия</string>
   </property>
  </action>
  <action name="circle">
   <property name="icon">
    <iconset>
     <normaloff>icons/инструменты/круг.png</normaloff>icons/инструменты/круг.png</iconset>
   </property>
   <property name="text">
    <string>Круг</string>
   </property>
  </action>
  <action name="red">
   <property name="icon">
    <iconset>
     <normaloff>icons/цвета/красный.png</normaloff>icons/цвета/красный.png</iconset>
   </property>
   <property name="text">
    <string>Красный</string>
   </property>
  </action>
  <action name="green">
   <property name="icon">
    <iconset>
     <normaloff>icons/цвета/зеленый.png</normaloff>icons/цвета/зеленый.png</iconset>
   </property>
   <property name="text">
    <string>Зеленый</string>
   </property>
  </action>
  <action name="blue">
   <property name="icon">
    <iconset>
     <normaloff>icons/цвета/синий.png</normaloff>icons/цвета/синий.png</iconset>
   </property>
   <property name="text">
    <string>Синий</string>
   </property>
  </action>
  <action name="small">
   <property name="icon">
    <iconset>
     <normaloff>icons/размеры/маленький.png</normaloff>icons/размеры/маленький.png</iconset>
   </property>
   <property name="text">
    <string>Маленький</string>
   </property>
  </action>
  <action name="med">
   <property name="icon">
    <iconset>
     <normaloff>icons/размеры/средний.png</normaloff>icons/размеры/средний.png</iconset>
   </property>
   <property name="text">
    <string>Средний</string>
   </property>
  </action>
  <action name="big">
   <property name="icon">
    <iconset>
     <normaloff>icons/размеры/большой.png</normaloff>icons/размеры/большой.png</iconset>
   </property>
   <property name="text">
    <string>Большой</string>
   </property>
  </action>
  <action name="orange">
   <property name="icon">
    <iconset>
     <normaloff>icons/цвета/оранжевый.png</normaloff>icons/цвета/оранжевый.png</iconset>
   </property>
   <property name="text">
    <string>Оранжевый</string>
   </property>
  </action>
  <action name="yellow">
   <property name="icon">
    <iconset>
     <normaloff>icons/цвета/желтый.png</normaloff>icons/цвета/желтый.png</iconset>
   </property>
   <property name="text">
    <string>Желтый</string>
   </property>
  </action>
  <action name="light_blue">
   <property name="icon">
    <iconset>
     <normaloff>icons/цвета/голубой.png</normaloff>icons/цвета/голубой.png</iconset>
   </property>
   <property name="text">
    <string>Голубой</string>
   </property>
  </action>
  <action name="action_9">
   <property name="text">
    <string>Синий</string>
   </property>
  </action>
  <action name="violet">
   <property name="icon">
    <iconset>
     <normaloff>icons/цвета/фиолетовый.png</normaloff>icons/цвета/фиолетовый.png</iconset>
   </property>
   <property name="text">
    <string>Фиолетовый</string>
   </property>
  </action>
  <action name="black">
   <property name="icon">
    <iconset>
     <normaloff>icons/цвета/черный.png</normaloff>icons/цвета/черный.png</iconset>
   </property>
   <property name="text">
    <string>Чёрный</string>
   </property>
  </action>
  <action name="white">
   <property name="icon">
    <iconset>
     <normaloff>icons/цвета/белый.png</normaloff>icons/цвета/белый.png</iconset>
   </property>
   <property name="text">
    <string>Белый</string>
   </property>
  </action>
  <action name="op">
   <property name="icon">
    <iconset>
     <normaloff>icons/работа с файлами/открыть.png</normaloff>icons/работа с файлами/открыть.png</iconset>
   </property>
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="sav">
   <property name="icon">
    <iconset>
     <normaloff>icons/работа с файлами/сохранить.png</normaloff>icons/работа с файлами/сохранить.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="saas">
   <property name="icon">
    <iconset>
     <normaloff>icons/работа с файлами/сохранить как.png</normaloff>icons/работа с файлами/сохранить как.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить как</string>
   </property>
  </action>
  <action name="color">
   <property name="icon">
    <iconset>
     <normaloff>icons/цвета/палитра.png</normaloff>icons/цвета/палитра.png</iconset>
   </property>
   <property name="text">
    <string>Палитра</string>
   </property>
  </action>
  <action name="pip">
   <property name="text">
    <string>Пипетка</string>
   </property>
  </action>
  <action name="pip_2">
   <property name="icon">
    <iconset>
     <normaloff>icons/инструменты/ластик.png</normaloff>icons/инструменты/ластик.png</iconset>
   </property>
   <property name="text">
    <string>Ластик</string>
   </property>
  </action>
  <action name="action_square">
   <property name="icon">
    <iconset>
     <normaloff>icons/инструменты/квадрат.png</normaloff>icons/инструменты/квадрат.png</iconset>
   </property>
   <property name="text">
    <string>Паралилипипед</string>
   </property>
  </action>
  <action name="grain">
   <property name="icon">
    <iconset>
     <normaloff>icons/инструменты/заливка.png</normaloff>icons/инструменты/заливка.png</iconset>
   </property>
   <property name="text">
    <string>Заливка</string>
   </property>
  </action>
  <action name="canv_col">
   <property name="icon">
    <iconset>
     <normaloff>icons/цвета/цвет фона.png</normaloff>icons/цвета/цвет фона.png</iconset>
   </property>
   <property name="text">
    <string>Цвет холста</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
